lib/matplotlib/tests/test_backend_bases.py

```diff
@@ -57,6 +57,7 @@ def test_get_default_filename(tmpdir):
 
 @pytest.mark.backend('pdf')
 def test_non_gui_warning():
+    plt.close('all')
```

Comment:
All figures should be closed by the fixture applied to all tests when they finish.

Reply:
Yeah, I was grasping at straws....